app/models/sipity.rb

```diff
@@ -57,23 +57,32 @@ def Agent(input, &block) # rubocop:disable Naming/MethodName
   # @return [Sipity::Entity]
   # rubocop:disable Naming/MethodName, Metrics/CyclomaticComplexity, Metrics/MethodLength
   def Entity(input, &block)
```

Comment:
let's prefer this inline, rather than in `fixme` (i doubt we will fix it)
```suggestion
  def Entity(input, &block) # rubocop:disable  Metrics/AbcSize
```

Reply:
makes sense

out of curiosity, why are we using a handwritten fixme file instead of rubocop's autogenerated?